### PR TITLE
Feature 2 physics and player controller script

### PR DESCRIPTION
**Pull request by JoyJazper**

### CODE REVIEW

File: Assets/Script/Physics/PlayerController.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+namespace PlayerPhysics
+{
+    [RequireComponent(typeof(Animator))]
+    public class PlayerController : MonoBehaviour
+    {
+        Animator MainCharecter;
+
+        float player_Speed;
+
+        Vector3 player_Scale;
```
Comment (JoyJazper) [inline Assets/Script/Physics/PlayerController.cs]: brought them out of update to stop the script from creating the variables on every frame.

File: Assets/Script/Physics/PlayerController.cs [hunk 2]
```diff
@@ -31,7 +59,39 @@ void Update()
...
 
+            // Player Jump motion section.
+            if (player_Vertical > 0)
+            {
+                mainCharecter_A.Play("Jump");
+
+                //ResizePlayerCollider(playerColliderScale_Jump, playerColliderOffset_Jump);
+            }
+            // We might need to use animations and add colliders in the jumping animation itself for perfection.
+            //else if (player_Vertical <= 0)
```
Comment (JoyJazper) [inline Assets/Script/Physics/PlayerController.cs]: Collider for Jump is not set up yet.

File: Assets/Script/Physics/PlayerController.cs [hunk 3]
```diff
@@ -0,0 +1,97 @@
+﻿using UnityEngine;
+
+namespace PlayerPhysics
+{
+    [RequireComponent(typeof(Animator), typeof(BoxCollider2D))]
+    public class PlayerController : MonoBehaviour
+    {
+        Animator mainCharecter_A;
```
Comment (mayankgrover) [inline Assets/Script/Physics/PlayerController.cs]: characterAnimator

File: Assets/Script/Physics/PlayerController.cs [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+            playerColliderOffset_Stand = mainCharecter_C.offset;
+
+            playerColliderScale_Couch = new Vector2(0.9175335f, 1.296851f);
+            playerColliderOffset_Couch = new Vector2(-0.1247323f, 0.6194392f);
+
+            //playerColliderScale_Jump = new Vector2(0.8660495f, 1.317989f);
+            //playerColliderOffset_Jump = new Vector2(0.05765058f, 1.778642f);
+        }
+
+        void Update()
```
Comment (mayankgrover) [inline Assets/Script/Physics/PlayerController.cs]: atleast 4 small function

**Review by mayankgrover — CHANGES_REQUESTED** [review]

**Pull request merged**